boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -64,6 +65,14 @@
 			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
 		};
 	};
+
+	flexcan2_phy: can-phy-tja1057 {
```

Comment:
Also, this does not follow the de-facto standard naming for CAN transceiver devicetree nodes.

```suggestion
	can_phy0: can-phy0 {
```